packages/utils/crates/utils-log/README.md

```diff
@@ -3,12 +3,44 @@
 The purpose of the `utils-log` Rust crate is to create a thin Rust WASM-compatible wrapper
 over the [log](https://docs.rs/log/latest/log/) library.
 
-The goal of this wrapper is to hide the internal switching mechanism for WASM and non-WASM related environments. This crate has no meaning for the JavaScript code, as it is a Rust code layer only.
+This crate also implements a `JsLogger` backend, which is inserted when a crate is loaded in the WASM context.
+Otherwise, the crate's default logging backend will be used (such as `env_logger`).
 
 ### Usage in Rust code
 
 Add this crate as a dependency and `use` the desired macros.
 
+#### Installing `JsLogger` backend in WASM context
+
+The `JsLogger` backend should be installed as soon as the crate is loaded in the WASM context.
+A suitable place would be the `xyz_set_panic_hook` in crate's `lib.rs` as this function is typically called immediately
+after the WASM binary has been loaded.
+
+Here's an example:
+
+```rust
+use utils_log::logger::JsLogger;
+
+static LOGGER: JsLogger = JsLogger {};
+
+#[allow(dead_code)]
+#[wasm_bindgen]
+pub fn my_crate_set_panic_hook() {
```

Comment:
This should then be called something like `initialize_wasm_for_my_crate()`

Reply:
Should we rename the functions in all crates then?